.github/workflows/posix-deps-apt.sh

```diff
@@ -15,6 +15,7 @@ apt-get -yq install \
     libgdbm-dev \
     libgdbm-compat-dev \
     liblzma-dev \
+    libmpdec-dev \
```

Comment:
@hugovk: FYI, this will fail for Ubuntu 24.04, as `libmpdec-dev` is unavailable there.

Reply:
I've been trying to figure out *why* that is, and have come up short (I get lost quickly in the Debian/Ubuntu development process).  All I've found is a note from @doko42 in the Ubuntu [`python3.11 3.11.2-4`](https://launchpad.net/ubuntu/+source/python3.11/3.11.2-4) changelog, saying "Build with internal mpdecimal library, so that mpdecimal can be removed for bookworm." (and indeed, Debian bookworm does not have a libmpdec package either).

Reply:
Found it:

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1013395